kura/org.eclipse.kura.util/src/main/java/org/eclipse/kura/util/base/ThrowableUtil.java

```diff
@@ -31,9 +32,11 @@ private ThrowableUtil() {
      *
      * @param throwable
      *            the throwable to propagate
+     * @throws RuntimeException
+     *             translates the provided exception as a runtime exception
      */
     public static void propagate(final Throwable throwable) {
-        throw new KuraRuntimeException(KuraErrorCode.INTERNAL_ERROR, throwable);
+        throw new RuntimeException(throwable);
```

Comment:
Wouldn't it be flexible to extract it as a helper method so that it can be used in other places too? Even I am using it for different Driver implementations of Kura Wires (Modbus and OPC-UA Driver).